### ISSUE TITLE
加载网络apng动画资源只显示静态图不能播放动画，加载本地资源没有问题

### ISSUE THREAD

**Issue by sun55** [penfeizhou/APNG4Android#54]
### New Issue Checklist

- [ ] I have searched for a similar issue in the [project](https://github.com/penfeizhou/APNG4Android/issues) and found none

### Issue Info

| Info                         | Value                                                 |      |
| ---------------------------- | ----------------------------------------------------- | ---- |
| Device Info                  | e.g. Huawei Mate 20                               |      |
| System Version               | e.g.  9.0                                             |      |
| APNG4Android Library Version | e.g.  1.3.1                                           |      |
| Repro rate                   | e.g. all the time (100%) / sometimes x% / only once   |      |
| Repro with our demo project  | e.g. does it happen with our demo project?            |      |
| Demo project link            | e.g. link to a demo project that highlights the issue |      |

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.
加载网络apng动画资源只显示静态图不能播放动画，加载本地资源没有问题

**Comment by jingpeng** [comment]
can you provide the url of the apng?

**Comment by sun55** [comment]
> can you provide the url of the apng?

gradle中的依赖是：
dependencies {
    implementation fileTree(dir: 'libs', include: ['*.jar'])
    implementation 'com.android.support:appcompat-v7:28.0.0'
    implementation 'com.android.support.constraint:constraint-layout:1.1.3'
    implementation 'com.android.support:recyclerview-v7:28.0.0'
    implementation 'com.github.bumptech.glide:glide:4.9.0'
    implementation 'com.github.bumptech.glide:annotations:4.9.0'
    annotationProcessor 'com.github.bumptech.glide:compiler:4.9.0'

    testImplementation 'junit:junit:4.12'
    androidTestImplementation 'com.android.support.test:runner:1.0.2'
    androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'

    implementation 'com.github.penfeizhou.android.animation:apng:1.3.1'
    implementation 'com.github.penfeizhou.android.animation:glide-plugin:1.3.1'
}

加载apng网络资源的函数是：
Glide.with(imageView).load("https://misc.aotu.io/ONE-SUNDAY/SteamEngine.png").into(imageView);

**Comment by jingpeng** [comment]
I tested your code & it totally works fine
